lib/grape/middleware/logger.rb

```diff
@@ -80,6 +81,10 @@ def after_failure(error)
     after(error[:status])
   end
 
+  def headers
+    env['grape.request.headers']
```

Comment:
Although, we should use the grape constants for the hash key (see `#parameters`)